spec/support/docs_whitelist.txt

```diff
@@ -0,0 +1,174 @@
+RuboCop::Cop::Lint::AssignmentInCondition
```

Comment:
Would "undocumented_cops" be a better name for this file? I didn't really get its purpose the first time I glanced at this PR.